dockstore-webservice/pom.xml

```diff
@@ -945,6 +945,29 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>1.6.0</version>
+                <executions>
+                    <execution>
+                        <id>swagger2openapi)</id>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <!--So it executes after the swagger.yaml's compile-->
+                        <phase>test</phase>
```

Comment:
Not really a test, should probably be in a different phase. 
Note that Maven runs plugins in the same phase in the order that they appear in the pom.xml

Reply:
interesting